ports/servo/browser.rs

```diff
@@ -302,6 +302,12 @@ impl Browser {
                 EmbedderMsg::LoadComplete => {
                     self.loading_state = Some(LoadingState::Loaded);
                 }
+                EmbedderMsg::CloseBrowser => {
+                    self.browser_id = None;
+                    // Nothing left to do for now,
+                    // but could hide a tab, and show another one, instead of quitting.
+                    self.event_queue.push(WindowEvent::Quit);
+                }
```

Comment:
Just found the `,` is inconsistent in this pattern matching.
Should we fix it in this PR or have an `E-easy` issue to make the consistency of comma for pattern matching?

Reply:
Thanks, I can add a comma for this specific change, and you could make it consistent across the rest in a separate issue?